test/seqn.lisp

```diff
@@ -1 +1,23 @@
 (in-package :geb-test)
+
+(define-test geb-seqn :parent geb-test-suite)
+
+(define-test seqn-evaluates-and-correctly
+  :parent geb-seqn
+  (is obj-equalp (list 1 0) (gapply (geb.trans:to-seqn geb-bool:and) (list 1 0 1 0)))
+  (is obj-equalp (list 0 0) (gapply (geb.trans:to-seqn geb-bool:and) (list 1 0 0 0)))
+  (is obj-equalp (list 0 0) (gapply (geb.trans:to-seqn geb-bool:and) (list 0 0 1 0)))
+  (is obj-equalp (list 0 0) (gapply (geb.trans:to-seqn geb-bool:and) (list 0 0 0 0))))
+
+(define-test seqn-de-compose
+  :parent geb-seqn
+  (is obj-equalp (list 1 0) (gapply (geb.seqn.spec:seqn-decompose 3) (list 4)))
+  (is obj-equalp (list 4) (gapply (geb.seqn.spec:seqn-concat 1 2) (list 1 0)))
+  (is obj-equalp (list 4) (gapply (geb.seqn.spec:composition (geb.seqn.spec:seqn-concat 1 2)
+                                                             (geb.seqn.spec:seqn-decompose 3))
+                                  (list 4)))
+  (is obj-equalp (list 1 0) (gapply (geb.seqn.spec:composition (geb.seqn.spec:seqn-decompose 3)
+                                                               (geb.seqn.spec:seqn-concat 1 2))
+                                  (list 1 0))))
+
```

Comment:
alias your seqn so this is just seqn: and remove the seqn words on your functions they only make the API ugly